tools/travis.ps1

```diff
@@ -270,32 +272,44 @@ else
             }            
         }
 
+    # if the tests did not pass, throw the reason why
+    if ( $result -eq "FAIL" ) {
+        Throw $resultError
+    }
+}
+elseif($Stage -in 'Failure', 'Success')
+{
+    $result = 'PASS'
+    if($Stage -eq 'Failure')
+    {
+        $result = 'FAIL'
+    }
+
+    if ((-not $isPr) -and $cronBuild) {
```

Comment:
Is `-not $isPr` necessary? `$cronBuild` is true only if `$env:TRAVIS_EVENT_TYPE -eq 'cron'`.

Reply:
resolved.